### PR TITLE
Electron Localization

### PR DESCRIPTION
**Pull request by dwalintukan**
Tried a couple of libs, but nothing worked well. Most node libs are piggybacked on Express.

Decided to just create a simple localization util class.

### CODE REVIEW

File: src/localization/i18n.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+const _ = require('lodash');
+
+const Utils = require('../utils/utils');
+const logger = require('../utils/logger');
+const locale = require('./locale.json');
+
+class i18n {
+  constructor() {
+    // Set locale based on OS language
+    let lang = 'en';
```
Comment (frankobe) [inline src/localization/i18n.js]: the default is `zh` according to frontend

File: src/localization/i18n.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    const osLang = Utils.getOsLanguage();
+    if (osLang) {
+      if (osLang.includes('en')) {
```
Comment (frankobe) [inline src/localization/i18n.js]: `if (osLang. startsWith('en')) `

File: src/localization/i18n.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        lang = 'en';
+      } else if (osLang.includes('zh')) {
```
Comment (frankobe) [inline src/localization/i18n.js]: if (osLang. startsWith('zh'))

File: src/localization/locale.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
+{
+  "cancel": {
+    "en": "Cancel"
```
Comment (frankobe) [inline src/localization/locale.json]: "zh": "取消"

File: src/localization/locale.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  },
+  "error": {
+    "en": "Error"
```
Comment (frankobe) [inline src/localization/locale.json]: "zh": "错误"

File: src/localization/locale.json [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  },
+  "functionDisabledUntilInitialized": {
+    "en": "This function is disabled until initializing is finished."
```
Comment (frankobe) [inline src/localization/locale.json]: "zh": "初始化完成后方能使用次功能"

File: src/localization/locale.json [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+  },
+  "launch": {
+    "en": "Launch"
```
Comment (frankobe) [inline src/localization/locale.json]: "zh": "启动"

File: src/localization/locale.json [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+  },
+  "mainnet": {
+    "en": "Mainnet",
+    "zh": "主链"
+  },
+  "ok": {
+    "en": "Ok"
```
Comment (frankobe) [inline src/localization/locale.json]: "zh": "OK"

Comment (frankobe) [inline src/localization/locale.json]: "Ok" -> "OK"?

File: src/localization/locale.json [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+  },
+  "qtumWalletDialogTitle": {
+    "en": "Launch Qtum Wallet?"
```
Comment (frankobe) [inline src/localization/locale.json]: "zh": "启动量子钱包?"

File: src/localization/locale.json [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+  },
+  "qtumWalletDialogMessage": {
+    "en": "You are about to launch the Qtum Wallet. Bodhi will need to be shutdown to run Qtum Wallet. To run Bodhi again, you will have to close the Qtum Wallet and run Bodhi again.\n\nBodhi will shutdown shortly. Are you sure to want to continue?"
```
Comment (frankobe) [inline src/localization/locale.json]: "zh": "菩提即将关闭已启动量子钱包。再次启动菩提前，请关闭量子钱包。\n\n菩提即将关闭，请确认是否继续"

File: src/localization/i18n.js (hunk 1 above)
Comment (dwalintukan) [reply src/localization/i18n.js]: done

File: src/localization/i18n.js (hunk 2 above)
Comment (dwalintukan) [reply src/localization/i18n.js]: done

File: src/localization/i18n.js (hunk 3 above)
Comment (dwalintukan) [reply src/localization/i18n.js]: done

File: src/localization/locale.json (hunk 4 above)
Comment (dwalintukan) [reply src/localization/locale.json]: added

File: src/localization/locale.json (hunk 5 above)
Comment (dwalintukan) [reply src/localization/locale.json]: added

File: src/localization/locale.json (hunk 6 above)
Comment (dwalintukan) [reply src/localization/locale.json]: added

File: src/localization/locale.json (hunk 7 above)
Comment (dwalintukan) [reply src/localization/locale.json]: addded

File: src/localization/locale.json (hunk 8 above)
Comment (dwalintukan) [reply src/localization/locale.json]: added

Comment (dwalintukan) [reply src/localization/locale.json]: fixed

File: src/localization/locale.json (hunk 9 above)
Comment (dwalintukan) [reply src/localization/locale.json]: added

File: src/localization/locale.json (hunk 10 above)
Comment (dwalintukan) [reply src/localization/locale.json]: added

**Pull request merged**